site/content/en/v1.4/api/extension_types.md

```diff
@@ -3437,6 +3437,16 @@ _Appears in:_
 | `text` | _string_ |  false  |  | Text defines the text accesslog format, following Envoy accesslog formatting,<br />It's required when the format type is "Text".<br />Envoy [command operators](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#command-operators) may be used in the format.<br />The [format string documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#config-access-log-format-strings) provides more information. |
 | `json` | _object (keys:string, values:string)_ |  false  |  | JSON is additional attributes that describe the specific event occurrence.<br />Structured format for the envoy access logs. Envoy [command operators](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#command-operators)<br />can be used as values for fields within the Struct.<br />It's required when the format type is "JSON". |
 
+**Note:**  
+`%PATH%` is **not a supported command operator** in Envoy for access log format strings.  
+To log the request path, use `%REQ(:path)%` instead.  
+See the [Envoy command operator documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#command-operators) for all valid options.
+
+**Example:**
+```
+%REQ(:method)% %REQ(:path)% %PROTOCOL%
+```
+
```

Comment:
this API doc is auto-generated, you shouldn't edit it directly.

I don't think we should point out what's not supported(it's endless), we just need to the users what's support.

Reply:
Highting what's support in https://github.com/envoyproxy/gateway/blob/edf2540741b2a5916de1727e0a367a5a31766555/site/content/en/latest/tasks/observability/proxy-accesslog.md